modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -9,24 +9,18 @@ endif::[]
 You can see a history of changes you make to a project in Flow Designer.
 The saved states are listed by the ID of the user who made the change, time, date, and activity. The activity name is based on the action you performed at save time.
```

Comment:
Sorry, but isn't this obvious: "The activity name is based on the action you performed at save time."? Or do you mean the name of the saved state?

Reply:
I do not think it is obvious. Activity is one type of a saved state. I thought it helpful to clarify what activity meant in this context. But if you feel that part is easy for the user to understand, then I defer to you.